### ISSUE TITLE
ResultReuse Support

### ISSUE THREAD

**Issue by Obsecurus** [uber/athenadriver#50]
Consider adding support for https://docs.aws.amazon.com/athena/latest/ug/reusing-query-results.html similar to https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/athena@v1.23.2/types#ResultReuseConfiguration

**Comment by This-Is-Not-Brad** [comment]
Agreed, I was just looking for this functionality. This would be really useful.
